lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -553,9 +554,8 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
     course = get_course_by_id(course_id)
     cohorts_header = ['Cohort Name'] if course.is_cohorted else []
 
-    partition_service = LmsPartitionService(user=None, course_id=course_id)
-    partitions = partition_service.course_partitions
-    group_configs_header = ['Experiment Group ({})'.format(partition.name) for partition in partitions]
+    experiment_partitions = get_split_user_partitions(course.user_partitions)
+    group_configs_header = ['Experiment Group ({})'.format(partition.name) for partition in experiment_partitions]
```

Comment:
Nice catch, @adampalay. Since I don't see that unicode is validated against in any of our code, I'll make this handle unicode.